packages/image_picker/test/image_picker_test.dart

```diff
@@ -34,11 +34,13 @@ void main() {
               'source': 0,
               'maxWidth': null,
               'maxHeight': null,
+              'imageQuality': 100
```

Comment:
also add a test for imageQuality is null after making imageQuality an optional param

Reply:
i've added the test for null condition